oura-library/src/main/kotlin/org/radarbase/oura/request/OuraRequestGenerator.kt

```diff
@@ -151,7 +143,9 @@ constructor(
                     request.user,
                     request.endDate,
                 )
-                userNextRequest[request.user.versionedId] = Instant.now().plus(ONE_DAY)
+                userNextRequest[request.user.versionedId] = Instant.now().plus(BACK_OFF_TIME)
```

Comment:
is this change intentional?

Reply:
Ah yes because I was thinking this would affect the other routes as well in case those routes have data. Is it fine?

Reply:
ah i see, perhaps then we should have per route per user next request map. But can be handled in a separate issue+PR. Thanks

Reply:
Ok sure, I've added here: https://github.com/RADAR-base/RADAR-REST-Connector/issues/149